x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -353,7 +421,110 @@ export const AlertsList: React.FunctionComponent = () => {
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
-
+      <EuiSpacer size="m" />
+      {!dissmissAlertErrors && alertsStatusesTotal[AlertExecutionStatusValues[2]] > 0 ? (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiCallOut
+              color="danger"
+              size="s"
+              title={
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.attentionBannerTitle"
+                  defaultMessage="There is {totalStausesError} {totalStausesError, plural, one {{singleTitle}} other {# {multipleTitle}}} with an error."
+                  values={{
+                    totalStausesError: alertsStatusesTotal[AlertExecutionStatusValues[2]],
+                    singleTitle: 'alert',
+                    multipleTitle: 'alerts',
+                  }}
+                />
+              }
+              iconType="alert"
+            >
+              <EuiButton
+                type="primary"
+                size="s"
+                color="danger"
+                onClick={() => setAlertStatusesFilter([AlertExecutionStatusValues[2]])}
+              >
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.viewBunnerButtonLabel"
+                  defaultMessage="View"
+                />
```

Comment:
When I select 'View', the alerts list is filtered to the alerts with error status, but the status filter does not show any selected filters.

In this screenshot, I have 1 OK Alert and 1 Error Alert. I have selected 'View' which filters to the Error Alert but the Status filter shows 0 selections, making it difficult to remove to filter
<img width="1179" alt="Screen Shot 2020-10-02 at 9 33 16 AM" src="https://user-images.githubusercontent.com/13104637/94929768-8996d800-0493-11eb-8aa2-636ab29f8e98.png">

Reply:
I'm seeing the same.  I'd be happy to have this be fixed later - I don't think we see too many errors in the alerts  themselves, so this isn't likely going to be a frequently used path anyway.  It is SOOOO awesome we're showing the error message on these tho in the alert details!